src/components/statusFilter.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+import { twJoin } from "tailwind-merge";
+
+interface OptionStyle {
+  option: string;
+  status: string;
+  activeClass: string;
+  inactiveClass: string;
+}
+
+interface StatusFilterProps {
+  onStatusChange: (status: string) => void;
+}
+
+export function StatusFilter({ onStatusChange }: StatusFilterProps) {
+  const [activeOption, setActiveOption] = useState("Todos");
+
+  const handleOptionClick = (optionStyle: OptionStyle) => {
+    setActiveOption(optionStyle.option);
+    onStatusChange(optionStyle.status);
+  };
+
+  const BASE_CLASSES = `py-2 px-4 rounded-lg focus:outline-none transition-colors duration-300`;
+
+  const optionStyles = [
```

Comment:
As it is a constant, I suggest you change the name to be uppercase. Also, optionStyles isn't a good name, because it manages more than styles, and they are basically the option configuration. I suggest something like:
```suggestion
   const OPTIONS= [
```

Reply:
yeah i agree with you